### ISSUE TITLE
cmake warnings re: Policy CMP0053

### ISSUE THREAD

**Issue by tjelbert** [annulen/webkit#296]
Policy CMP0053 is not set: Simplify variable reference and escape sequence
  evaluation.  Run "cmake --help-policy CMP0053" for policy details.  Use the
  cmake_policy command to set the policy and suppress this warning.

  For input:

```
'
Name: @PKGCONFIG_TARGET_LIBNAME@
Version: @PROJECT_VERSION@
Libs: -L@CMAKE_INSTALL_PREFIX@/@EGPF_LIB_INSTALL_DIR@ -l@PKGCONFIG_TARGET_LIBNAME@
Cflags: @PKGCONFIG_TARGET_INCLUDES@ @PKGCONFIG_TARGET_DEFINES@
Requires: @PKGCONFIG_TARGET_DEPS@
'
```

  the old evaluation rules produce:

```
'
Name: Qt5WebKit
Version: 5.602.1
Libs: -L/opt/qt5/lib -lQt5WebKit
Cflags: -I/opt/qt5/include/Qt5WebKit
Requires: Qt5Core Qt5Gui Qt5Network
'
```

  but the new evaluation rules produce:

```
'
Name: @PKGCONFIG_TARGET_LIBNAME@
Version: @PROJECT_VERSION@
Libs: -L@CMAKE_INSTALL_PREFIX@/@EGPF_LIB_INSTALL_DIR@ -l@PKGCONFIG_TARGET_LIBNAME@
Cflags: @PKGCONFIG_TARGET_INCLUDES@ @PKGCONFIG_TARGET_DEFINES@
Requires: @PKGCONFIG_TARGET_DEPS@
'
```

  Using the old result for compatibility since the policy is not set.
Call Stack (most recent call first):
  Source/WebKit/PlatformQt.cmake:384 (ecm_generate_pkgconfig_file)
  Source/cmake/WebKitMacros.cmake:5 (include)
  Source/WebKit/CMakeLists.txt:50 (WEBKIT_INCLUDE_CONFIG_FILES_IF_EXISTS)

**Comment by annulen** [comment]
This looks strange to me - we declare `cmake_minimum_required(VERSION 2.8.12)` and AFAIK it should set all policy settings to match the behavior of 2.8.12

**Comment by annulen** [comment]
Fixed in 2fdc3a8ca642d0a16f9722165960faac388935b9
